frontend/src/views/OfferView.vue

```diff
@@ -1,24 +1,28 @@
 <script setup lang="ts">
+import Button from "primevue/button";
 import Calendar from "primevue/calendar";
 import Card from "primevue/card";
 import Dropdown from "primevue/dropdown";
 import FloatLabel from "primevue/floatlabel";
+import Textarea from "primevue/textarea";
 import { onMounted, ref } from "vue";
 import { useRoute } from "vue-router";
 
 import { getOrder } from "@/backendClient";
+import OfferItem from "@/components/orders/OfferItem.vue";
 import { OfferStatus } from "@/global/types/appTypes";
+import { ArticleKind } from "@/global/types/appTypes";
 import type { OfferCreate } from "@/global/types/dataEditTypes";
-import type { Offer } from "@/global/types/entities";
-import type { Order } from "@/global/types/entities";
+import type { Offer, Order } from "@/global/types/entities";
 import { ROUTES } from "@/router";
 
+let id = 1;
```

Comment:
I don't think we need this ID in order to create an offer 🤔

Reply:
We need unique id for v-for. I won't send it to the backend.

Reply:
Makes sense. Can we rename it to something like: "itemCount"?